content/execution_model.tex

```diff
@@ -52,3 +52,11 @@ \subsection{Progress of OpenSHMEM Operations}\label{subsec:progress}
       provide asynchronous one-sided operations, as these have very limited
       performance value for \openshmem programs.
 \end{itemize}
+
+\subsection{Invoking OpenSHMEM Operations}
+
+Pointer arguments to \openshmem routines that point to non-const data must not
+overlap in memory with other arguments to the same \openshmem operation.
+Otherwise, the behavior is undefined.  Two arguments overlap in memory if any
+of their data elements are located at the same address on the same \ac{PE}.
+
```

Comment:
Add: Some OpenSHMEM routines have multiple symmetric object arguments.  Unless noted, OpenSHMEM routines do not requires symmetric objects to be of the same kind, e.g. static versus dynamically allocated symmetric objects.

Reply:
@nspark Pointed out that the above doesn't adequately deal with shmem_ptr.